### DISCUSSION TITLE
how to create glass effect navbar in tailwind css

### DISCUSSION THREAD

**Discussion by pragatheeswarancse** [tailwindlabs/tailwindcss#3132] [Ideas]
[https://vercel.com/home?utm_source=next-site&utm_medium=banner&utm_campaign=next-website](url)

like this

**Comment by raunometsa** [comment]
I think you need a plugin for this, like: https://github.com/benface/tailwindcss-filters

**Comment by mainhurt** [comment]
Dont need plugins for it..

`                    <div class="flex justify-center align-center close">
                <div class="close absolute top-3 left-3 flex w-80 backdrop-filter backdrop-blur-lg bg-gradient-to-r from-green-400 opacity-90 to-amber-400 border-t-4 border-teal-500 rounded-b text-teal-900 px-4 py-3 shadow-md" role="alert">
                        <div class="py-1 "><svg xmlns="http://www.w3.org/2000/svg" class="h-10 w-10 m-3 bg-" viewBox="0 0 20 20" fill="currentColor">
  <path fill-rule="evenodd" d="M16.707 5.293a1 1 0 010 1.414l-8 8a1 1 0 01-1.414 0l-4-4a1 1 0 011.414-1.414L8 12.586l7.293-7.293a1 1 0 011.414 0z" clip-rule="evenodd" />
</svg></div>
                        <div>`

I use it code:
 backdrop-filter backdrop-blur-lg  opacity-90 
bg-gradient-to-r from-green-400 to-amber-400

**Comment by pragatheeswarancse** [comment]
Thank you, it really helps

**Comment by VicGUTT** [comment]
You can also take a look at [headlessui](https://headlessui.dev/)'s navbar or this [quick demo](https://play.tailwindcss.com/L8eBLi1fk3). ✌🏾
